lib/workers/global/limits.ts

```diff
@@ -37,3 +38,164 @@ export function isLimitReached(key: Limit): boolean {
   const { max, current } = limit;
   return max - current <= 0;
 }
+
+export type CountName = 'PullRequests' | 'HourlyPullRequests' | 'Branches';
```

Comment:
```suggestion
export type CountName = 'ConcurrentPRs' | 'HourlyPRs' | 'Branches';
```

Make consistent, and shorter